index.js

```diff
@@ -1,7 +1,7 @@
 const Kucoin = require('./src/api'),
     config = require('./config');
 
-const lang = require('./src/lang/' + config.lang);
+const lang = require('./src/lang/' + config.ui.lang);
```

Comment:
You can merge the const variable definition of lang with the others :) It'll work anyway.

Reply:
Done in https://github.com/demirhancosku/kucoin-api-nodejs/pull/1/commits/8ef3115e3ee111d64b7198fa41a0da2fdcc67223 :)